lib/storage/s3.ts

```diff
@@ -55,6 +58,11 @@ const MIN_STORED_ID_LENGTH = 9;
 
 assert(MIN_STORED_ID_LENGTH >= PREFIX_LENGTH, 'MIN_STORED_ID_LENGTH must be at least PREFIX_LENGTH');
 
+export type TestReq = {
```

Comment:
oh boy :)

Reply:
Yeah.. more of those will be needed on the way for full TSification.